Dockerfile

```
@@ -1,8 +1,17 @@
+FROM pandoc/typst:3.7.0 AS build_pdf
+COPY ./src ./src
+COPY ./theme ./theme
+COPY ./typst ./typst
+COPY ./typst_compile.sh ./typst_compile.sh
+RUN ./typst_compile.sh
```

Räcker det inte att bara kopiera theme när vi bygger med mdbook?
```suggestion
FROM pandoc/typst:3.7.0 AS build_pdf
COPY ./src ./src
COPY ./typst ./typst
COPY ./typst_compile.sh ./typst_compile.sh
RUN ./typst_compile.sh
```

---

Done